lib/chef-dk/skeletons/code_generator/files/default/build_cookbook/README.md

```diff
@@ -10,9 +10,9 @@ In the parent directory to this build_cookbook, the `config.json` can be modifie
 
 ## Test Kitchen - Local Verify Testing
 
-This cookbook also has a `.kitchen.yml` which can be used to create local build nodes with Test Kitchen to perform the verification phases, `unit`, `syntax`, and `lint`. When running `kitchen converge`, the instances will be set up like Chef Delivery "build nodes" with the [delivery_build cookbook](https://github.com/chef-cookbooks/delivery_build). The reason for this is to make sure that the same exact kind of nodes are used by this build cookbook are run on the local workstation as would run Delivery. It will run `delivery job verify PHASE` for the parent project.
+This cookbook also has a `kitchen.yml` which can be used to create local build nodes with Test Kitchen to perform the verification phases, `unit`, `syntax`, and `lint`. When running `kitchen converge`, the instances will be set up like Chef Delivery "build nodes" with the [delivery_build cookbook](https://github.com/chef-cookbooks/delivery_build). The reason for this is to make sure that the same exact kind of nodes are used by this build cookbook are run on the local workstation as would run Delivery. It will run `delivery job verify PHASE` for the parent project.
```

Comment:
> The reason for this is to make sure that the same exact kind of nodes are used by this build cookbook are run on the local workstation as would run Delivery.

Suggest: 

> This ensures that the local Kitchen nodes are configured to look the same as Delivery build nodes, providing consistent results for the verification phases. 

Related: should this be Automate and not Delivery?

Reply:
I think more correctly it would be Workflow specifically, not Automate

Reply:
I updated this to Workflow and updated a few other mistakes I noticed.